local/README.md

```diff
@@ -68,6 +68,8 @@ Note that all these properties have their corresponding environment variables th
   - secretAccessKey - AWS_SECRET_ACCESS_KEY
  - S3 - configuration related to S3
   - bucket - AWS_S3_BUCKET
+ - dynamo - dynamodb config
+  - tablename - the dynamodb tablename to use.  must have a key column named `Id`
```

Comment:
Can we please bit more details here? I mean the purpose of the table which is to store webhook payloads as temporary store.

Reply:
Yep.  I went back and forth on whether we wanted the app to create the table or have it already created, and I figured it would be better if the table was created as part of the setup and not the app itself.

My thinking was that the table could be kept around even after the app is killed, or restarted, or redeployed.  That way we can see which webhooks failed to process and might involve some human intervention (or another offline process that attempts reconciliation).

Reply:
Understood. My concern is just to add some details in the README about what is the purpose of the table.

Reply:
Ah, I see.  Ya, could definitely add more information in the next PR.